### PR TITLE
Add documentation about scrubbing params

### PR DESCRIPTION
**Pull request by mjquinlan2000**
Adding documentation per #50

### CODE REVIEW

File: pages/Using Rollbax in Plug-based applications.md [hunk 1]
```diff
@@ -50,4 +50,20 @@ defp handle_errors(conn, %{kind: kind, reason: reason, stack: stacktrace}) do
 end
 ```
 
+#### Sensitive Data
```
Comment (whatyouhide) [inline pages/Using Rollbax in Plug-based applications.md]: Can we move this section at the very end, after the "Check the documentation for the Rollbar API [...]" part?

File: pages/Using Rollbax in Plug-based applications.md [hunk 2]
```diff
@@ -50,4 +50,20 @@ defp handle_errors(conn, %{kind: kind, reason: reason, stack: stacktrace}) do
...
+
+Due to the fact that ALL parameters are available to the controller, there is a possibility that sensitive data such as `password`/`password_confirmation` could be sent to Rollbar. The params should be scrubbed for any sensitive data before a request is made to Rollbar.
```
Comment (whatyouhide) [inline pages/Using Rollbax in Plug-based applications.md]: * No need to emphasize "ALL" IMO, wdyt? :)

**Comment by whatyouhide** [comment]
Actually @mjquinlan2000, I decided to just merge and fix the very minor nitpicks myself so that I don't cause you any more trouble 😄 Thanks a lot, great work (and great catch)! 💟

**Comment by mjquinlan2000** [comment]
Lol @whatyouhide no worries. I figured you would have some input on how the documentation should be worded

**Pull request merged**